app/src/main/java/com/github/swent/echo/compose/components/searchmenu/SearchMenuFilters.kt

```diff
@@ -134,7 +134,7 @@ fun SearchMenuFilters(
                     Dropdown(
                         "Sort By",
                         SortBy.entries.map { stringResource(it.stringKey) },
-                        filters.sortBy?.ordinal ?: -1,
+                        filters.sortBy.ordinal,
```

Comment:
Question: Any particular reason as you to why you removed the null pointer check?

Reply:
Nathan changed it so it's not nullable anymore, so there's no need for a null check